packages/vm/package.json

```diff
@@ -101,16 +101,5 @@
   "bugs": {
     "url": "https://github.com/ethereumjs/ethereumjs-vm/issues"
   },
-  "homepage": "https://github.com/ethereumjs/ethereumjs-vm",
-  "nyc": {
-    "exclude": [
-      "tests"
-    ]
-  },
-  "standard": {
```

Comment:
Removing the `standard` key here since we are exclusively using the `ethereumjs-config` linter now.